shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1423,4 +1460,67 @@ - (BOOL)isPresentingViewController {
   return self.presentedViewController != nil || self.isPresentingViewControllerAnimating;
 }
 
+- (flutter::PointerData)generatePointerDataForMouse API_AVAILABLE(ios(13.4)) {
+  flutter::PointerData pointer_data;
+
+  pointer_data.Clear();
+
+  pointer_data.kind = flutter::PointerData::DeviceKind::kMouse;
+  pointer_data.change = _mouseState.flutter_state_is_added ? flutter::PointerData::Change::kAdd
+                                                           : flutter::PointerData::Change::kHover;
+  pointer_data.pointer_identifier = reinterpret_cast<int64_t>(_pointerInteraction.get());
+
+  pointer_data.physical_x = _mouseState.location.x;
+  pointer_data.physical_y = _mouseState.location.y;
+
+  return pointer_data;
+}
+
+- (UIPointerRegion*)pointerInteraction:(UIPointerInteraction*)interaction
+                      regionForRequest:(UIPointerRegionRequest*)request
+                         defaultRegion:(UIPointerRegion*)defaultRegion API_AVAILABLE(ios(13.4)) {
+  if (request != nil) {
+    auto packet = std::make_unique<flutter::PointerDataPacket>(1);
+    const CGFloat scale = [UIScreen mainScreen].scale;
+    _mouseState.location = {request.location.x * scale, request.location.y * scale};
+
+    flutter::PointerData pointer_data = [self generatePointerDataForMouse];
+
+    pointer_data.signal_kind = flutter::PointerData::SignalKind::kNone;
+    packet->SetPointerData(0 /* index */, pointer_data);
```

Comment:
The proper style for argument comments is `packet->SetPointerData(/*index=*/0, pointer_data);`  If you use that format they can be verified by static analysis.  https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments